tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/BuildDummyPackages.cs

```diff
@@ -60,6 +60,8 @@ class EmptyPackageFile : IPackageFile
 
             public FrameworkName TargetFramework => new FrameworkName("netstandard", new Version(1,0,0));
 
+            public DateTimeOffset LastWriteTime => DateTimeOffset.Now;
```

Comment:
FWIW, here's the NuGet commit that added this to the interface. The tagging matches up with the version change: https://github.com/NuGet/NuGet.Client/commit/9b8c7a834e276b4f3211096787cc7d53e52744c0.